rts/lra/lra-test/pom.xml

```diff
@@ -75,7 +75,8 @@
                 </property>
             </activation>
             <properties>
-                <lra.coordinator.trace.params>-Dthorntail.logging=TRACE</lra.coordinator.trace.params>
+<!--                <lra.coordinator.trace.params>-Dthorntail.logging=TRACE</lra.coordinator.trace.params>-->
```

Comment:
Could you please remove this line here? I think it's not necessary here anymore when the coordinator runs as a quarkus application.

Reply:
Okay, I will remove this.